python-sdk/nuscenes/predict/helper.py

```diff
@@ -0,0 +1,195 @@
+# nuScenes dev-kit.
+# Code written by Freddy Boulton, 2020.
+
+from typing import Dict, Tuple, Any, List, Callable
+from pyquaternion import Quaternion
+from nuscenes import NuScenes
+from nuscenes.utils.geometry_utils import transform_matrix
+from nuscenes.eval.common.utils import quaternion_yaw
+import numpy as np
+
+MICROSECONDS_PER_SECOND = 1e6
+BUFFER = 0.15 # seconds
+
+def angle_of_rotation(yaw: float) -> float:
+    """Given a yaw (measured from x axis) find the angle needed to rotate by so that
+    the yaw points upwards (pi / 2)."""
```

Comment:
Yaw is in the xy plane. How can it point upwards?

Reply:
Updated the comment. Upwards = aligned with y-xais :sweat_smile: